generators/E906LegacyGen/SQPrimaryVertexGen.C

```diff
@@ -16,34 +16,39 @@ from Kun to E1039 experiment in Fun4All framework
 
 #include "SQPrimaryVertexGen.h"
 //#include "SQBeamlineObject.h"
-
-SQPrimaryVertexGen::SQPrimaryVertexGen() 
+double beam_vtx_x; 
+double beam_vtx_y;
+ TF2* beam_global;
```

Comment:
I will delete the beam_vtx_x and beam_vtx_y (Thank you)

I defined TF2* beam_global so that I can pass the profile set from Fun4Sim to https://github.com/abinashpun/e1039-core/blob/c357feb85eb1fbb188cd5c3b078b7b2852a8ee28/generators/E906LegacyGen/SQPrimaryVertexGen.C#L39-L46
and can be passed to another function generateVtxPerp() as 
https://github.com/abinashpun/e1039-core/blob/c357feb85eb1fbb188cd5c3b078b7b2852a8ee28/generators/E906LegacyGen/SQPrimaryVertexGen.C#L286-L293

Reply:
The two blocks that you mentioned are in the member functions.  Thus a member variable (beamProfile or _beam_profile) should be sufficient.

Reply:
I tried different ways not to use any global variable. I tried it in that way too but could not make it work. Maybe I can check again with other ideas.